src/BlendingProvider/Composition/ViewModels/CompositionInfo.json.cs

```diff
@@ -2,11 +2,35 @@
 using Starcounter;
 using Starcounter.Advanced.XSON;
 using BlendingCommon;
+using BlendingProvider.Website.ViewModels;
 
 namespace BlendingProvider.Composition.ViewModels
 {
     partial class CompositionInfo : Json
     {
+        public bool IsCurrentScopeDirty
+        {
+            get
+            {
+                var currentValue = false;
+                if (this.AttachedScope != null)
```

Comment:
Please remove redundant nesting level.

```suggestion
                if (this.AttachedScope == null)
                {
                    return false;
                }
```